libs/ipc/inc/public/tfc/ipc/details/filter.hpp

```diff
@@ -247,6 +248,11 @@ struct any_filter_decl<std::string> {
   using value_t = std::string;
   using type = std::variant<filter<filter_e::filter_out, value_t>>;
 };
+template <>
+struct any_filter_decl<ipc::details::mass_t> {
+  using value_t = ipc::details::mass_t;
+  using type = std::variant<filter<filter_e::filter_out, value_t>>;
```

Comment:
This is how I would like this. 

But I am not sure whether adding this filter would actually work, can you test it? If doesn't work open a followup issue.

Reply:
Added a test for the filter, it works.